### PR TITLE
Fixed dropdown issues in admin/events#show page

### LINKED ISSUE #1172: admin/events#show dropdown does not close
In admin/events#show the dropdowns for difficulty_level and track do not close when clicking elsewhere.

Does anyone else experience that behavior?

I tried in chromium & firefox on my laptop which runs openSUSE Leap. I see the same behavior in https://osem-demo-pr-1114.herokuapp.com/admin/conference/osemdemo/program/events/2

### PR DESCRIPTION
**Pull request by siddhantbajaj**
closes #1172

### CODE REVIEW

File: app/views/admin/events/_proposal.html.haml [hunk 1]
```diff
@@ -17,7 +17,7 @@
           %b Type
         %td
           .dropdown
-            = link_to '#', class: 'dropdown-toggle', id: 'type-dropdown' do
+            = link_to '#', class: 'dropdown-toggle', id: 'type-dropdown','data-toggle'=>'dropdown' do
```
Comment (differentreality) [inline app/views/admin/events/_proposal.html.haml]: It would be awesome if you can also add spaces:
* after the comma
* before =>
* after =>

Comment (siddhantbajaj) [inline app/views/admin/events/_proposal.html.haml]: done.

**Comment by differentreality** [comment]
:tada: Awesome :tada:
Please squash your commits into 1 for merging!

**Comment by siddhantbajaj** [comment]
done :+1:

**Comment by differentreality** [comment]
:+1: great, thanks!

**Pull request merged**